pkg/asset/installconfig/nutanix/validation.go

```diff
@@ -53,6 +38,14 @@ func ValidateForProvisioning(ic *types.InstallConfig) error {
 		return field.InternalError(field.NewPath("platform", "nutanix"), errors.Wrapf(err, "unable to connect to Prism Central %q", p.PrismCentral.Endpoint.Address))
 	}
 
+	// validate whether a prism element with the UUID actually exists
+	for _, pe := range p.PrismElements {
+		_, err = nc.V3.GetCluster(pe.UUID)
+		if err != nil {
+			return field.InternalError(field.NewPath("platform", "nutanix", "prismElements"), errors.Wrapf(err, "prism element UUID %s does not correspond to a valid prism element in Prism", pe.UUID))
+		}
+	}
+
```

Comment:
Now that you getting the "Name" of the PrismElement as an optional parameter, do you need to validate if that exists also? And I am assuming that the Name should correspond to the same UUID as within the Nutanix cluster?

Reply:
Yes, the name should match the UUID, it is being validated in `create cluster` phase - or filled in if its unset.